### PR TITLE
Sheets

### LINKED ISSUE #4: Hook up Google Sheets
Title is self-explanatory

### PR DESCRIPTION
**Pull request by rchhong**
Fixes #4.

### CODE REVIEW

File: sheets.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+        # Call the Sheets API
+        sheet = service.spreadsheets()
+        result = sheet.values().get(spreadsheetId=SPREADSHEET_ID,
+                                    range=URL_RANGE).execute()
+        values = result.get('values', [])
+
+        if not values:
+            print('No data found.')
+            return
```
Comment (thebunanna) [inline sheets.py]: Should return {} or else you get an exception if there's no values in sheet.

File: sheets.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        with open('token.json', 'w') as token:
+            token.write(creds.to_json())
+
+    try:
+        service = build('sheets', 'v4', credentials=creds)
+
+        # Call the Sheets API
+        values = [
+            [
+                url, title, f"{circle_name} ({author_name})", genre, is_r18, price_in_yen
```
Comment (thebunanna) [inline sheets.py]: might as well convert to USD here and have a commadn to reconvert later if needed?

**Review by thebunanna — CHANGES_REQUESTED** [review]

File: sheets.py (hunk 2 above)
Comment (rchhong) [reply sheets.py]: This was an artifact when the changes from main weren't merged.

File: sheets.py (hunk 1 above)
Comment (rchhong) [reply sheets.py]: Fixed in new commit

File: sheets.py (hunk 2 above)
Comment (rchhong) [reply sheets.py]: Fixed in new commit

**Review by thebunanna — APPROVED** [review]

**Pull request merged**